app/services/action_queue.rb

```diff
@@ -155,6 +155,15 @@ def transaction_payload
       }
     end
 
+    def fake_card_info
+      {
+        card_num:       "4111111111111111",
```

Comment:
@osahyoun I know this is merged but for Paypal aren't we passing PYPL as the last 4? Perhaps we should be doing something similar hear instead so we don't have fake numbers that look like real numbers in AK.